resources/assets/less/bem/contest-voting-list.less

```diff
@@ -0,0 +1,127 @@
+/**
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License version 3
+*    as published by the Free Software Foundation.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+
+.contest-voting-list {
+  @top: contest-entry-list;
+  &__table {
+    width: 100%;
+    max-width: 600px;
+    margin: 0 auto;
+  }
+
+  &__row {
+    background-color: #293336;
+    color: #8dacb6;
+    font-weight: 100;
+    font-size: 12px;
+    border-bottom: 2px solid #1A2223;
+    transition: 0.1s;
+    display: flex;
+
+    &:hover {
+      background-color: lighten(#293336, 2%);
+    }
+
+    &--selected {
+      background-color: rgba(141, 172, 182, 0.25);
+    }
+
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+
+  &__header {
+    text-transform: uppercase;
+    color: #8dacb6;
+    font-size: 10px;
+    line-height: 32px;
+    display: flex;
+  }
+
+  &__header-wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__icon {
+    flex: 0 0 32px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    &--bg {
+      background: rgba(0, 0, 0, 0.75);
+    }
+  }
+
+  &__header-title {
+    padding-left: 10px;
+  }
+
+  &__header-votesummary {
+    display: flex;
+    height: 31px;
+    align-items: center;
+  }
+
+  &__preview {
+    background-color: rgba(0,0,0,0.75);
+    width: 32px;
+    height: 32px;
+  }
+
+  &__title {
+    color: white;
+    padding: @spacing;
+    transform: translate(0);
+    overflow: hidden;
+
+    &--show-votes {
+      align-items: baseline;
+      width: 100%;
+      padding: 2px @spacing;
+      height: 32px;
+      background: transparent url(/images/layout/vote-1x1.png) 0 center repeat-y;
+      flex-direction: column;
+      justify-content: space-around;
+    }
+
+    &--inner {
```

Comment:
doesn't seem needed? (parent is flex column stretch aligned)

<sub>also, not bem</sub>